src/main/java/edu/unc/lib/boxc/migration/cdm/services/CdmIndexService.java

```diff
@@ -335,6 +355,43 @@ private String indexFieldType(String exportField) {
         }
     }
 
+    /**
+     * Indexes all exported objects for this project
+     * @throws IOException
+     */
+    public void indexAllFromCsv(CdmIndexOptions options) throws IOException {
+        assertObjectsExported();
+
+        CdmFieldInfo fieldInfo = fieldService.loadFromCsvFieldsFromProject(project);
+        List<String> exportFields = fieldInfo.listAllExportFields();
+        recordInsertSqlTemplate = makeInsertTemplate(exportFields);
+
+        try (
+                var conn = openDbConnection();
+                var csvParser = ExportObjectsService.openMappingsParser(Path.of(options.getCsvFile()));
```

Comment:
The CSV file is not going to be exactly like the export file format. See my comment below about the number of columns. So I think we just want to parse the provided CSV file as a generic CSV file rather than as an exported_objects.csv. The purpose of having the user export and then import a CSV file is so that they can add more info per record than we get from just the file system (which at the moment is basically just file paths).

If we need to provide a list of header fields, that would probably be `exportFields`.